packages/dotcom-ui-base-styles/package.json

```diff
@@ -27,11 +27,10 @@
   },
   "devDependencies": {
     "check-engine": "^1.10.1",
-    "react": "^16.8.6",
```

Comment:
Oh removed as we stipulate npm 7 or greater, and because npm does install peerDependencies with npm install now :)

https://github.blog/2020-10-13-presenting-v7-0-0-of-the-npm-cli/

Reply:
i'm aware npm 7 installs peerDependencies, but i thought that only happened when running `npm install package` but not when running `npm install` in the package. turns out i'm wrong about that